File: benchmark/results/ts-script-sonnet/run-004/a-mcp/output/src/domain/notification/services/delivery-dispatcher.ts
import { Notification } from '../aggregates/notification';

/** DeliveryDispatcher service. */
export class DeliveryDispatcher {

  create(input: Partial<Notification>): Notification { throw new Error('not implemented'); }

  findById(id: string): Notification | null { throw new Error('not implemented'); }

  list(limit: number): Notification[] { throw new Error('not implemented'); }

  delete(id: string): void { throw new Error('not implemented'); }
}

File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/notification/services/DeliveryDispatcher.java
package com.metaengine.demo.notification.services;

import com.metaengine.demo.notification.aggregates.Notification;
import java.util.List;

/** Domain service that dispatches notifications to the relevant channels. */
public class DeliveryDispatcher {
    /** Dispatch a notification from the provided partial input. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Look up a dispatched notification by id. Returns null when not found. */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} dispatched notifications. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Cancel dispatch for the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
